kivy/lang.py

```diff
@@ -1795,6 +1795,10 @@ def match(self, widget):
             parents[cls] = classes
         return self.key in parents[cls]
 
+    def match_widget_name(self, widget_name):
+        return self.key == widget_name.lower()
+
+
 
```

Comment:
Sorry, I had an issue with the hook so I ran it manually but forgot for the second commit..